### DISCUSSION TITLE
Greating

### DISCUSSION THREAD

**Discussion by N5348** [community/community#32296] [New to GitHub] [answered]
Hello 
Am Ntakirutimana from Rwanda 
Am new learner of coding
I have some vocabulary in this  course not understand well
For example:
Tags , repository, release. Please hp me to understand well
Thanks

**Comment by airtower-luna** [comment, marked as answer]
The first two are *Git* terms. Git is a system for version control. A repository holds the versioned code (or other data), a tag marks a specific version of some importance. I recommend you look at the [Git Book](https://git-scm.com/book/en/v2) site for details. Note that there are translations in various languages, look at the left sidebar.

A "release" usually means a version of software that's ready for general use. In Git a release version is usually marked by a tag with the version number.

**Comment by N5348** [comment]
Thanks for your helper for me
